### PR TITLE
My hw 03

### PR DESCRIPTION
**Pull request by agodes84**

### CODE REVIEW

**Comment by vladhilko** [comment]
RSpec is red

File: HW03/andrei_godes/andrei_godes_hw_03_t_03.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'time'
+class Homework3
+  def task3(logs)
+    array = subtraction(include_string(logs))
```
Comment (B0RlS) [inline HW03/andrei_godes/andrei_godes_hw_03_t_03.rb:6]: Could you please give more descriptive variable names?
The 'array' doesn't say anything about what's in there (it applies to everything else too)
Try to give descriptive names to variables that will reflect what you put in them

File: HW03/andrei_godes/andrei_godes_hw_03_t_04.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task4(logs)
+    let = logs.count('A-z')
```
Comment (B0RlS) [inline HW03/andrei_godes/andrei_godes_hw_03_t_04.rb:5]:
```suggestion
    let = logs.count('a-zA-Z')
```

**Review by B0RlS — APPROVED** [review]
You are great, clean solutions and clear code, keep up the good work

**Pull request merged**